aip/general/0121.md

```diff
@@ -81,6 +81,27 @@ In an API with a stateless protocol, the server has the responsibility for
 persisting data, which may be shared between multiple clients, while clients
 have sole responsibility and authority for maintaining the application state.
 
+### Cyclic References
+
+The relationship between resources, such as parent-child or
+[resource references][], **must** be representable via an acylic graph.
+
+A cyclic relationship between resources increases the complexity of managing
+resources significantly more complex. Consider a resource A and B that refer to
+each other. The process to create said resources are:
+
+1. create resource A without a reference to B. Retrieve id for resource A.
+2. create resource B with a reference to A. Retrieve id for resource B.
+3. update resource A with the reference to B.
+
+The delete operation may also become more complex, due to reasoning about which resource must be dereferenced first for a successful deletion.
```

Comment:
Nit: Can you limit this line to 80 characters?

Reply:
done!